src/block.ml

```diff
@@ -125,6 +137,10 @@ module Pre = struct
             s
         in
         {blocks; next = Rfenced_code (ind, num, q, info, Sub.to_string s :: lines, a)}
+    | Rdef_list (term, d::defs), Lparagraph ->
+        {blocks; next = Rdef_list (term, (d ^ "\n" ^ (Sub.to_string s))::defs)}
```

Comment:
Can you say a few words about this case?

Reply:
This case is what adds the support for multiline definitions:
If we find a paragraph line after an Rdef_list, we append it to the last definition.
I'll add an example